pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -81,13 +86,18 @@ spec:
             - "--leader-election-renew-deadline={{ .LeaderElectionRenewDeadline }}"
             - "--leader-election-retry-period={{ .LeaderElectionRetryPeriod }}"
             - "--default-fstype=ext4"
+            - "--logtostderr=false"
+            - "--alsologtostderr=true"
+            - "--log_file=/var/lib/cephcsi/log/rbd-provisioner/csi-attacher.log"
```

Comment:
The last info I heard, the CSI project sidecar images didn't have log flags. Is that not true? Maybe I missed something

Reply:
CSI project sidecar was supported after I tested on the live cluster